ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/ModifyUser.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3cli.command;
+
+import com.google.common.collect.ImmutableMap;
+import com.spectralogic.ds3cli.Arguments;
+import com.spectralogic.ds3cli.exceptions.CommandException;
+import com.spectralogic.ds3cli.models.GetDataPoliciesResult;
+import com.spectralogic.ds3cli.models.GetUsersResult;
+import com.spectralogic.ds3cli.util.Ds3Provider;
+import com.spectralogic.ds3cli.util.FileUtils;
+import com.spectralogic.ds3client.commands.spectrads3.*;
+import com.spectralogic.ds3client.models.*;
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import com.spectralogic.ds3client.utils.SSLSetupException;
+import org.apache.commons.cli.MissingOptionException;
+
+import java.io.IOException;
+import java.security.SignatureException;
+
+public class ModifyUser extends CliCommand<GetUsersResult> {
+
+    // name or guid
+    private String userId;
+    private String defaultPolicyId;
+    private ImmutableMap<String, String> metadata;
+
+
+    public ModifyUser(final Ds3Provider provider, final FileUtils fileUtils) {
+        super(provider, fileUtils);
+    }
+
+    @Override
+    public CliCommand init(final Arguments args) throws Exception {
+        this.userId = args.getId();
+        if (this.userId == null) {
+            throw new MissingOptionException("The alter policy command requires '-i' to be set with the username or Id");
+        }
+        this.metadata = args.getMetadata();
```

Comment:
At this point we only have the one attribute that can be modified so let's change it. If there are more we can update it then, but we wouldn't use the metadata attribute for that anyway.